### DISCUSSION TITLE
I meet `ProviderError: Must be authenticated!` when I try `yarn deploy`

### DISCUSSION THREAD

**Discussion by ChainDev931105** [PaulRBerg/hardhat-template#92] [Q&A] [answered]
when I try `yarn deploy`,
I meet this error
`ProviderError: Must be authenticated!`

**Comment by PaulRBerg** [comment, marked as answer]
You did not set a valid Infura API key. Please create a `.env` file by following the provided [.env.example](https://github.com/paulrberg/solidity-template/blob/main/.env.example) file.
